.changes/unreleased/Docs-20240108-193257.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Docs
+body: license it's not showing in pypi page
```

Comment:
We use changie documents for generating our changelog at release time. With that in mind, we want the message from them to say what was fixed, instead of what was wrong. With that in mind, I think we want to update the `body` here to be something more like `Ensured license information gets displayed in pypi`.

Reply:
perfect I change it, thanks for the feedback